reference/7.1/Microsoft.PowerShell.Core/About/about_PowerShell_NuGet_Packages.md

```diff
@@ -0,0 +1,202 @@
+---
+keywords: powershell,cmdlet,nuget
+ms.date: 09/27/2019
+online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_pipelines?view=powershell-7.x&WT.mc_id=ps-gethelp
+schema: 2.0.0
+title: about_Pipelines
```

Comment:
The URL and the title refer to about_Pipelines.

Reply:
Ah that's where I copied the header from